lua/entities/gmod_wire_expression2/core/custom/ftrace.lua

```diff
@@ -1147,7 +1484,7 @@ e2function number ftrace:getSurfaceFlags()
 end
 
 __e2setcost(3)
-e2function number ftrace:getDisplayFlags()
+e2function number ftrace:getDisplaceFlags()

```

Comment:
DisplaceFlags?

Reply:
[Exactly, that's why I renamed it](https://wiki.facepunch.com/gmod/Structures/TraceResult). The previews one was very confusing:

```lua
DispFlags [number]
The displacement flags of the hit surface. See Enums/DISPSURF.
```

Reply:
Make the name getDisplacementFlags

Reply:
I would like it to be a bit shorter...

Reply:
DisplaceFlags is a terrible name though

Reply:
How about `getDispFlags` ? the same as the trace result field?